sourcegraph/cody
# Edit: Add more telemetry specifics

## Description

This PR improves telemetry specifics to, when available, include:
- The LLM used
- The intent of edit (e.g. fix, test, doc)
- The mode of the edit (e.g. edit, insert)
- The source of the edit (e.g. code-action, menu)

## Test plan

1. Make edits (e.g. fix, doc, instruction-based, test, generate)
2. Confirm the telemetry matches the config of the edit
